### DISCUSSION TITLE
How to set the release_mode in audio?

### DISCUSSION THREAD

**Discussion by devapaine** [flet-dev/flet#2147] [Q&A] [answered]
Hey,

Can anyone provide an example of how to set the release_mode in audio?

I've tried:

    audio1 = ft.Audio(
        src="https://luan.xyz/files/audio/ambient_c_motion.mp3", release_mode=ft.ReleaseMode.STOP,
    )
    page.overlay.append(audio1)

and 

audio1.release_mode.STOP()

Thanks

**Comment by ndonkoHenri** [comment]
The first.

**Reply by devapaine** [reply]
Hi @ndonkoHenri,
    src=f"/images/error.mp3", autoplay=False, release_mode=ft.ReleaseMode.STOP,
                                                           ^^^^^^^^^^^^^^
AttributeError: module 'flet' has no attribute 'ReleaseMode'

Am I missing something?

Using v0.13.0

**Reply by ndonkoHenri** [reply]
You are right, I opened an issue to solve that: https://github.com/flet-dev/flet/issues/2155

**Reply by ndonkoHenri** [reply, marked as answer]
Ah! You have to do `ft.audio.ReleaseMode` instead. 😄

**Reply by devapaine** [reply]
That's the one. Thank you!
